src/main/java/uk/gov/hmcts/reform/civil/service/StateGeneratorService.java

```diff
@@ -106,4 +105,20 @@ && isRespondentsResponseSatisfied(caseData, caseData.toBuilder().build())) {
         return AWAITING_ADDITIONAL_INFORMATION;
     }
 
+    private static boolean hasPayment(CaseData caseData) {
+        return caseData.getGeneralAppPBADetails().getAdditionalPaymentDetails() != null;
+    }
+
+    private static boolean isConsentOrderRespondentSatisfied(CaseData caseData) {
+        return nonNull(caseData.getGeneralAppConsentOrder()) && isRespondentsResponseSatisfied(
+            caseData,
+            caseData.toBuilder().build()
+        );
+    }
+
+    private boolean isUrgentWithoutNotice(CaseData caseData) {
+        return caseData.getGeneralAppInformOtherParty() != null
+            && YesOrNo.NO.equals(caseData.getGeneralAppInformOtherParty().getIsWithNotice())
```

Comment:
might extend this and put the null check  for caseData.getGeneralAppInformOtherParty() != null before checking. getIsWithNotice()